src/controllers.php

```diff
@@ -163,6 +164,10 @@
         $link->setAttribute('class', 'anchor');
         $node->appendChild($link);
 
+        if (!isset($first_title) && strlen($title) > 0) {
```

Comment:
@GrahamCampbell empty will work too... I guess there is a length check in the template, so we could remove `strlen($title) > 0` too... my only thought is that an incorrect documentation page might stop a valid title getting passed to the template.

I will push a new commit now dropping the strlen and renaming the variable.